CollapseLauncher/Classes/InstallManagement/BaseClass/InstallManagerBase.cs

```diff
@@ -1,12 +1,16 @@
+using CollapseLauncher.Extension;
```

Comment:
(CodeQA) usings order.
Tbh we can defer this fix later if you dont wanna since I planned to do another full codeQA checks after this gets merged

Reply:
Should be resolved